BD-3349: Updates to translation API docs (#7862)
braze-inc/braze-docs: _docs/_api/endpoints/translations/get_bulk_translations_canvases.md

| Parameter | Required | Data Type | Description |
| --------- | ---------| --------- | ----------- |
|`step_id`| Required | String | The ID of your Canvas step. |
|`message_variation_id`| Required | String | The ID of your message. |
{: .reset-td-br-1 .reset-td-br-2 .reset-td-br-3  .reset-td-br-4}

> Note all translation IDs are considered universal unique identifiers (UUIDs), which can be in **Multi-Language Support** settings. You can also find the UUID in the request response.

Author: ```suggestion
Note all translation IDs are considered universal unique identifiers (UUIDs), which can be found in **Multi-Language Support** settings or in the request response.
```

| Parameter | Required | Data Type | Description |
| --------- | ---------| --------- | ----------- |

> |`canvas_id`| Required | String | The ID of your Canvas. |

Reviewer: ```suggestion
|`step_id`| Required | String | The ID of your Canvas step. |
```
